src/contacts/ViewContact/ViewContactContainer.js

```diff
@@ -24,15 +27,16 @@ class ViewContactContainer extends Component {
     resources: PropTypes.object,
     baseUrl: PropTypes.string.isRequired,
     mutator: PropTypes.object,
-    // eslint-disable-next-line react/no-unused-prop-types
     orgId: PropTypes.string,
     showMessage: PropTypes.func.isRequired,
-    match: PropTypes.object,
+    match: ReactRouterPropTypes.match.isRequired,
+    history: ReactRouterPropTypes.history.isRequired,
+    backToOrgDetails: PropTypes.bool,
```

Comment:
IMHO, it should be `onClose` or something general, and it should be a callback.

Reply:
Fixed